### ISSUE TITLE
NPM published package contains sample/node_modules folder

### ISSUE THREAD

**Issue by melfa** [progre/tslint-config-airbnb#30]
Install from NPM:
```
$ npm install tslint-config-airbnb
$ du -h -d4
 12K	./node_modules/tslint-microsoft-contrib/tests
200K	./node_modules/tslint-microsoft-contrib/utils/implicitRoles
 48K	./node_modules/tslint-microsoft-contrib/utils/attributes
320K	./node_modules/tslint-microsoft-contrib/utils
928K	./node_modules/tslint-microsoft-contrib
252K	./node_modules/tslint-consistent-codestyle/rules
 12K	./node_modules/tslint-consistent-codestyle/src
292K	./node_modules/tslint-consistent-codestyle
 20K	./node_modules/tslib/docs
 88K	./node_modules/tslib
 76M	./node_modules/tslint-config-airbnb/sample/node_modules
 76M	./node_modules/tslint-config-airbnb/sample
 76M	./node_modules/tslint-config-airbnb
 36K	./node_modules/esutils/lib
 52K	./node_modules/esutils
 72K	./node_modules/doctrine/lib
104K	./node_modules/doctrine
224K	./node_modules/tsutils/util
 88K	./node_modules/tsutils/typeguard
348K	./node_modules/tsutils
4,0K	./node_modules/isarray/build
 20K	./node_modules/isarray
4,0K	./node_modules/tslint-eslint-rules/dist/support
484K	./node_modules/tslint-eslint-rules/dist/rules
488K	./node_modules/tslint-eslint-rules/dist
144K	./node_modules/tslint-eslint-rules/node_modules/tsutils
144K	./node_modules/tslint-eslint-rules/node_modules
704K	./node_modules/tslint-eslint-rules
 79M	./node_modules
 79M	.
```
Notice total size and size of node_modules/tslint-config-airbnb/sample/node_modules folder.

Install from github:
```
$ npm install git+ssh://git@github.com/progre/tslint-config-airbnb#5.6.0
$ du -h -d4
 12K	./node_modules/tslint-microsoft-contrib/tests
200K	./node_modules/tslint-microsoft-contrib/utils/implicitRoles
 48K	./node_modules/tslint-microsoft-contrib/utils/attributes
320K	./node_modules/tslint-microsoft-contrib/utils
928K	./node_modules/tslint-microsoft-contrib
252K	./node_modules/tslint-consistent-codestyle/rules
 12K	./node_modules/tslint-consistent-codestyle/src
292K	./node_modules/tslint-consistent-codestyle
 20K	./node_modules/tslib/docs
 88K	./node_modules/tslib
 28K	./node_modules/tslint-config-airbnb/sample
100K	./node_modules/tslint-config-airbnb
 36K	./node_modules/esutils/lib
 52K	./node_modules/esutils
 72K	./node_modules/doctrine/lib
104K	./node_modules/doctrine
224K	./node_modules/tsutils/util
 88K	./node_modules/tsutils/typeguard
348K	./node_modules/tsutils
4,0K	./node_modules/isarray/build
 20K	./node_modules/isarray
4,0K	./node_modules/tslint-eslint-rules/dist/support
484K	./node_modules/tslint-eslint-rules/dist/rules
488K	./node_modules/tslint-eslint-rules/dist
144K	./node_modules/tslint-eslint-rules/node_modules/tsutils
144K	./node_modules/tslint-eslint-rules/node_modules
704K	./node_modules/tslint-eslint-rules
2,6M	./node_modules
2,6M	.
```
Notice total size and no folder node_modules/tslint-config-airbnb/sample/node_modules

**Comment by progre** [comment]
Oops.. thanks for your report.

**Comment by progre** [comment]
This was fixed in 5.7.0.
Thank you!
